es/lib/alpha.js

```diff
@@ -13,7 +13,7 @@ export var alpha = {
   'nn-NO': /^[A-ZÆØÅ]+$/i,
   'hu-HU': /^[A-ZÁÉÍÓÖŐÚÜŰ]+$/i,
   'pl-PL': /^[A-ZĄĆĘŚŁŃÓŻŹ]+$/i,
-  'pt-PT': /^[A-ZÃÁÀÂÇÉÊÍÕÓÔÚÜ]+$/i,
+  'pt-PT': /^[A-ZÃÁÀÂÄÇÉÊËÍÏÕÓÔÖÚÜ]+$/i,
```

Comment:
Unstage this unrelated changes.

Reply:
This one yes, just checked the portuguese alphabets, thanks for mentioning it 👍